oauth2client/client.py

```diff
@@ -279,8 +280,7 @@ def new_from_json(cls, s):
       An instance of the subclass of Credentials that was serialized with
       to_json().
```

Comment:
Should the "Args" section of this doc string be changed to better describe the input type as a byte string?